src/sql/parts/notebook/notebook.contribution.ts

```diff
@@ -59,15 +59,30 @@ const viewModelEditorDescriptor = new EditorDescriptor(
 Registry.as<IEditorRegistry>(EditorExtensions.Editors)
 	.registerEditor(viewModelEditorDescriptor, [new SyncDescriptor(NotebookInput)]);
 
-// todo: Will remove this code.
+// Feature flag for built-in Notebooks. Will be removed in the future.
+const configurationRegistry = Registry.as<IConfigurationRegistry>(ConfigExtensions.Configuration);
+configurationRegistry.registerConfiguration({
+	'id': 'notebook',
+	'title': 'Notebook',
+	'type': 'object',
+	'properties': {
+		'notebook.enabled': {
+			'type': 'boolean',
+			'default': false,
+			'description': localize('notebook.enabledDescription', 'Enable.')
```

Comment:
What does this look like? Can we change this to be a little more descriptive?

Reply:
Whoops. Will definitely fix.